src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1958,6 +1958,13 @@
     <Field Name="Value" Type="BoundExpression"/>
   </Node>
 
+  <Node Name="BoundUnconvertedCollectionArguments" Base="BoundNode">
```

Comment:
>BoundUnconvertedCollectionArguments [](http://example.com/codeflow?start=14&length=35)

nit: consider "BoundWithElement" since we already use the "...Element" naming convention for various collection expression elements (spread, key-value pair) and it would line up with the name of the syntax node ("WithElementSyntax")